### ISSUE TITLE
Rocket.Chat

### ISSUE THREAD

**Issue by lonix1** [twbs/icons#1586]
We already have icons for Teams, Slack and Discord.

[Rocket.Chat](https://www.rocket.chat) is an open-source and self-hosted alternative. It's massively popular. Please consider adding an icon for it?

Logo:
<img src="https://global-uploads.webflow.com/611a19b9853b7414a0f6b3f6/6132017f51d1acdee170210c_webclip%20256px.png" />

Fontawesome reference: https://fontawesome.com/icons/rocketchat?s=&f=brands

**Comment by korki43** [comment]
I don't think it is necessary to create a dedicated icon for this brand. The logo is already really minimalistic so you can use it next to Bootstrap icons without changing anything.
